src/main/scala/org/opennetworkinsight/DNSWordCreation.scala

```diff
@@ -20,13 +33,7 @@ import scala.math._
       var numparts = spliturl.length
       var domain = "None"
       var subdomain = "None"
-      //var country ="None"
```

Comment:
While writing unit testing I found there is no uses for host variable. Removed carefully.